CHANGELOG.md

```diff
@@ -4,10 +4,32 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## 1.11.2 - 2020-10-13
+## 1.12.0 - 2020-10-19
+**_Warning:_**
+- This update modifies the MongoDB schema. Make sure to start the application with `-DMONGOUPDATE=1`.
+- This update modifies information stored in Elasticsearch used for text based searching. Make sure to initiate a reindex 
+  of Elasticsearch from the Admin menu or by `POST /api/reindex`.
+
+### Added
+- Ability to set chunksize for clowder when downloading files. This is changed to 1MB from 8KB. This will result in 
+  faster downloads and less CPU usage at the cost of slightly more memory use.
+- Support for parsing of Date and Numeric data in new metadata fields. New search operators <, >, <=, >= have been 
+  added to search API now that those data are properly compared.
+- Global extractors page now shows more information, including submission metrics, logs (using Graylog), job history and
+  extractors maturity.
+- Users have more refined options to set extractors triggers at the space level. They can now follow global settings, 
+  disable and enable triggers.
+
+### Fixed
+- Remove secretKey from extractor log.
```

Comment:
not sure we need to mention that here since it is a fix to something that was never released.